data/src/main/java/com/android254/data/network/util/TokenProvider.kt

```diff
@@ -19,7 +19,7 @@ import kotlinx.coroutines.flow.Flow
 
 interface TokenProvider {
 
-    suspend fun fetch(): Flow<String?>
+    suspend fun fetch(): Flow<String?> // @Allan -> Don't suspend Flows
```

Comment:
Remove comment.

Reply:
Hey @chepsi 
This comment is just a heads up to @jumaallan so he can refactor this. @jumaallan please check this

Reply:
@anselmoalexandre saw it

I can raise a PR, but if you want, please feel free to refactor it here

Reply:
> @anselmoalexandre saw it
> 
> I can raise a PR, but if you want, please feel free to refactor it here

You can do the PR it's okay 👌🏿. Also, this was only heads up 😄